RESOURCES.md

```diff
@@ -0,0 +1,46 @@
+| [HackMD](https://hackmd.io/team/eiffel-community?nav=overview)                     | Emil Bäckmark                                                                                             |
+| [Travis CI](https://app.travis-ci.com/organizations/eiffel-community/repositories) | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |
+| [Codacy](https://app.codacy.com/)                                                  | ?                                                                                                         |
+| [Reviewable](https://reviewable.io/)                                               | ?                                                                                                         |
+
+## Google groups
+
+| Resource                                                                                                 | Main Contact                                                                                              |
+|----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| [eiffel-community](https://groups.google.com/g/eiffel-community)                                         | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |
+| [eiffel-tc](https://groups.google.com/g/eiffel-tc)                                                       | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |
+| [eiffel-community-security](mailto:eiffel-community-security@googlegroups.com)                           | [Security Officers](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#security-officers) |
+| [eiffel-protocol-maintainers](mailto:eiffel-protocol-maintainers@googlegroups.com)                       | ?                                                                                                         |                       
+| [eiffel-commons-maintainers](mailto:eiffel-commons-maintainers@googlegroups.com)                         | ?                                                                                                         |                         
+| [eiffel-easy2use-repository-maintainers](mailto:eiffel-easy2use-repository-maintainers@googlegroups.com) | ?                                                                                                         |
+| [etos-maintainers](mailto:etos-maintainers@googlegroups.com)                                             | Tobias Persson                                                                                                         |                                             

```

Comment:
```suggestion
| [etos-maintainers](mailto:etos-maintainers@googlegroups.com)                                             | [Tobias Persson](mailto:Tobias.Persson@axis.com)                                                                                                         |                                             

```

Reply:
Updated per TC notes in Updated in [8562e0b](https://github.com/eiffel-community/community/pull/179/commits/8562e0b96c4b8becdfefebcc7a7e06b3879a0f53)